moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -531,6 +531,8 @@ void PlanningSceneMonitor::newPlanningSceneCallback(const moveit_msgs::PlanningS
 
 void PlanningSceneMonitor::clearOctomap()
 {
+  if (scene_->getWorldNonConst()->removeObject(scene_->OCTOMAP_NS))
+    triggerSceneUpdateEvent(UPDATE_SCENE);
```

Comment:
I'm somewhat late to the party, but why is that `UPDATE_SCENE` instead of `UPDATE_GEOMETRY`? (we have been using that in our workaround code that should have been contributed for some while ...)